### PR TITLE
Rewrite some of the Ring of Chordata

### LINKED ISSUE #340: Two Rings of Chordata compound strangely
When you're at least one block above the water's floor, you go up quickly and with no way to stop it. When you enter the water, you fall down extremely fast.

### PR DESCRIPTION
**Pull request by howtonotwin**
Add guard against having two rings equipped; the second ring will be
inactive. Fixes #340

### CODE REVIEW

File: src/main/java/vazkii/botania/common/item/equipment/bauble/ItemWaterRing.java [hunk 1]
```diff
@@ -23,6 +24,9 @@
 
 public class ItemWaterRing extends ItemBauble implements IManaUsingItem {
 
+    private static final double SPEED_MULT = 1.2;
+    private static final double MAX_SPEED = 1.3;
```
Comment (yrsegal) [inline src/main/java/vazkii/botania/common/item/equipment/bauble/ItemWaterRing.java]: different indentation scheme than rest of the file

Comment (howtonotwin) [inline src/main/java/vazkii/botania/common/item/equipment/bauble/ItemWaterRing.java]: .-. Woops

**Comment by williewillus** [comment]
what're all of the other changes for if it simply disables a seocnd ring?

**Comment by yrsegal** [comment]
looks like just clarity refactors to me

**Comment by yrsegal** [comment]
also it calls things less times

**Pull request merged**